internal/provider/schema_gitlab_project_milestone.go

```diff
@@ -0,0 +1,121 @@
+package provider
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/xanzy/go-gitlab"
+)
+
+func gitlabProjectMilestoneGetSchema() map[string]*schema.Schema {
+	validMilestoneStates := []string{"active", "closed"}
+
+	return map[string]*schema.Schema{
```

Comment:
Let's put the arguments on top of the schema definition here for readability ...

Reply:
I don't pretty good understand what you mean. Could you clarify that?

Reply:
Oh sorry, basically all non-computed arguments starting with the required ones, then optional, then computed :)